app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\StorePostRequest;
+use App\Models\Post;
+use Image;
+
+class PostController extends Controller{
+ 
+    public function index(){
```

Comment:
Highly recommend checking out https://www.php-fig.org/psr/psr-12/ and getting your IDE to do this for you on save / reformat code.
```suggestion
class PostController extends Controller
{
    public function index()
    {
```